### PR TITLE
#1 Ux update fragments

### PR DESCRIPTION
**Pull request by huythien1708**

### CODE REVIEW

**Comment by huythien1708** [comment]
[Fix UI bug](https://trello.com/c/FbPfyZir/1-fix-ui-bug)

**Comment by huythien1708** [comment]
[Fix UI bug](https://trello.com/c/FbPfyZir/1-fix-ui-bug)

File: app/src/main/java/vn/realtest/stock/justanewproject/Data/FundStock.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        return stock_can_trade;
+    }
+
+    public void setStock_can_trade(String stock_can_trade) {
+        this.stock_can_trade = stock_can_trade;
+    }
+
+
+
+
```
Comment (namth2704) [inline app/src/main/java/vn/realtest/stock/justanewproject/Data/FundStock.java]: Please remove redundant empty lines

File: app/src/main/java/vn/realtest/stock/justanewproject/Adapter/MarketAdapter.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    @Override
+    public void onBindViewHolder(MarketStockViewHolder holder, int position) {
+        MarketStock marketStock = marketStockList.get(position);
+        holder.stock_name.setText(marketStock.getStock_name());
+        holder.stock_value.setText(marketStock.getStock_value());
+        holder.stock_change.setText(marketStock.getStock_change_rate());
+        holder.stock_vol.setText(marketStock.getStock_vol());
+        Log.d("abc", "String trả về: " + marketStock.getStock_change_rate());
```
Comment (namth2704) [inline app/src/main/java/vn/realtest/stock/justanewproject/Adapter/MarketAdapter.java]: Please remove all logs before commit

**Pull request merged**